feature-flags/feature-flags.production.ts

```diff
@@ -14,4 +14,5 @@ export const flagsProduction: FeatureFlagsInterface = {
   FEATURE_ACCOUNTS_SNAPSHOT: false,
   FEATURE_TEMPORARY_ACCOUNTS_SNAPSHOT_PAGE: false,
   FEATURE_ECOSYSTEM_ACTORS: false,
+  FEATURE_ECOSYSTEM_ACTORS_ABOUT: true,
```

Comment:
This page is under development so this should be set to false on production

Reply:
done